alert_group_nl_log2slack.py

```diff
@@ -124,7 +127,8 @@ def send_slack_message(message):
     print(f'sending: {data}')
     ret = requests.post(
         SLACK_WEBHOOK_URL, data=data, headers={
-            'Content-Type': 'application/json'})
+            'Content-Type': 'application/json'},
+            timeout=10)
     assert ret.status_code == 200, (ret, ret.text)
```

Comment:
Bad indentation, because it appears as though 'Content-Type' and `timeout` are on the same level.

Better alternative:
```python
    ret = requests.post(
        SLACK_WEBHOOK_URL, data=data,
        headers={'Content-Type': 'application/json'},
        timeout=10)
```